src/main/java/com/rocketden/main/service/UserService.java

```diff
@@ -37,6 +38,17 @@ public UserDto createUser(CreateUserRequest request) {
         return UserMapper.toDto(user);
     }
 
+    public boolean deleteUser(DeleteUserRequest request) {
+        User user = repository.findUserByNickname(request.getNickname());
+
+        if (user == null) {
+            return false;
+        }
+
+        repository.delete(user);
+        return true;
+    }
```

Comment:
Thoughts on returning error handling similar to the other functions instead of `true` / `false`?

Reply:
Done, per our discussion yesterday!